src/components/WeeklySummary/CurrentPromptModal.jsx

```diff
@@ -1,23 +1,65 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
 import { toast } from 'react-toastify';
 import ReactTooltip from 'react-tooltip';
 import { boxStyle } from 'styles';
+import { connect } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit } from '@fortawesome/free-regular-svg-icons';
+import { updateDashboardData, getDashboardDataAI } from '../../actions/dashboard';
+
+function CurrentPromptModal(props) {
+  const current = `Please edit the following summary of my week's work. Make sure it is professionally written in 3rd person format. Write it as only one paragraph. It must be only one paragraph. Keep it less than 500 words. Start the paragraph with 'This week'. Make sure the paragraph contains no links or URLs and write it in a tone that is matter-of-fact and without embellishment. Do not add flowery language, keep it simple and factual. Do not add a final summary sentence. Apply all this to the following:`;
+  const dashboardDataAIPrompt =
+    props.state.dashboardData === '' || props.state.dashboardData === null
+      ? current
+      : props.state.dashboardData;
```

Comment:
dashboardData comes as an object but we need only dashboardData.aIPromptText to set as a newPrompt. (now it crashes the app)
I would suggest to make it like this: 
`const dashboardDataAIPrompt = props.state.dashboardData.aIPromptText || current;
`

Reply:
Good catch.  Pushed my changes to address this issue.